scripts/mutect2_wdl/mutect2-dream-eval-sge.wdl

```diff
@@ -0,0 +1,191 @@
+# how to run
+# 1) start a cromwell server:
+# java -Dconfig.file=sge_pc_application_ruchi_cc.conf -jar $cromwell server
```

Comment:
I added the config file `sge.conf` to the repo. The identical copy of `sge.conf` is in `/humgen/gsa-hpprojects/dev/tsato/wdl/sge_pc_application_ruchi.conf`.